modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/AlterTableAlterColumnCommand.java

```diff
@@ -153,7 +152,8 @@ private void validateColumnChange(CatalogTableColumnDescriptor origin) {
             }
         }
 
-        if (precision != null && precision != origin.precision() && origin.type() != ColumnType.DECIMAL) {
+        if (precision != null && precision != origin.precision() && (origin.type() != ColumnType.DECIMAL
```

Comment:
why we had/have special case for DECIMAL type?

Reply:
i just add more strong restrictions for already existing ones.